shell/dialog/AddProjectMemberDialog.vue

```diff
@@ -79,7 +79,7 @@ export default {
   <Card class="prompt-rotate" :show-highlight-border="false" :sticky="true">
     <h4 slot="title" class="text-default-text" v-html="t('addProjectMemberDialog.title')" />
 
-    <div slot="body" class="pl-10 pr-10">
+    <div slot="body" class="pl-10 pr-10 pb-10">

```

Comment:
I think @nwmac [mentioned here](https://github.com/rancher/dashboard/issues/7101#issuecomment-1286255377) that we should replace the box shadow with the border.

Reply:
I didn't notice the requirement about border, thanks!